### PR TITLE
prometheus-operator upgrade 

### PR DESCRIPTION
**Pull request by kimschles**
## BACKGROUND

* This PR upgrades the version of the prometheus-operator 1-click app. 
-----------------------------------------------------------------------

## Changes
* This commit updates the helm chart version of the prometheus-operator to `12.11.13` which is the most recent version.

Information about that helm chart can be found here: https://artifacthub.io/packages/helm/mesosphere/prometheus-operator

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/linkerd2/deploy.sh [hunk 1]
```diff
@@ -1,38 +0,0 @@
-#!/bin/sh
```
Comment (paulanunda) [inline stacks/linkerd2/deploy.sh:1]: Did you mean to delete this?

**Review by paulanunda — CHANGES_REQUESTED** [review]

File: stacks/linkerd2/deploy.sh (hunk 1 above)
Comment (jgannondo) [reply stacks/linkerd2/deploy.sh:1]: @kimschles wanted to ping once more on this - please let us know re: Paul's comment. Thanks!

Comment (kimschles) [reply stacks/linkerd2/deploy.sh]: Sorry for the delay. That deletion was a mistake that I fixed in a new commit. Thanks for checking!

**Comment by paulanunda** [comment]
@kimschles given that prometheus-operator has been [deprecated](https://github.com/helm/charts/tree/master/stable/prometheus-operator) in favor of `kube-prometheus-stack`, I'm going to delist this standalone app. The [monitoring stack](https://marketplace.digitalocean.com/apps/kubernetes-monitoring-stack) will install `kube-prometheus-stack`.

**Pull request closed without merging**